Release article: 4.75 (#34390)
fleetdm/fleet: articles/fleet-4.75.0.md

# Fleet 4.75.0 | Omarchy Linux, Android configuration profiles, and more...

<div purpose="embedded-content">
   <iframe src="https://www.youtube.com/embed/a6qx4th3dKs?si=KYVIvqZTb9AZM27Y" frameborder="0" allowfullscreen></iframe>
</div>

Fleet 4.75.0 is now available. See the complete [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.75.0) or read on for highlights. For upgrade instructions, visit the [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.

## Highlights

- Omarchy Linux
- Android configuration profiles
- Smallstep certificates
- Labels page
- Easy-to-read MDM commands

> ### Omarchy Linux

Reviewer: ```suggestion
### Arch Linux / Omarchy Linux
```

> Fleet now supports [Omarchy](https://omarchy.org/) Linux. You can view host vitals like software inventory, run scripts, and install software.

Reviewer: ```suggestion
Fleet now supports [Arch Linux](https://archlinux.org/) and [Omarchy](https://omarchy.org/) Linux. You can view host vitals like software inventory, run scripts, and install software.
```

## Highlights

- Omarchy Linux
- Android configuration profiles
- Smallstep certificates
- Labels page
- Easy-to-read MDM commands

### Omarchy Linux

Fleet now supports [Omarchy](https://omarchy.org/) Linux. You can view host vitals like software inventory, run scripts, and install software.

### Android configuration profiles

You can now apply custom settings to work profiles on employee-owned (BYOD) Android hosts using configuration profiles. This lets you keep Android hosts compliant and secure. Learn how to create in [this video](TODO).

### Smallstep certificates

Fleet now integrates with [Smallstep](https://smallstep.com/) as a certificate authority. You can deliver Wi-Fi/VPN [SCEP](https://en.wikipedia.org/wiki/Simple_Certificate_Enrollment_Protocol) certificates to macOS, iOS, and iPadOS hosts to automate secure network access for your end users. Learn more in [the guide](https://fleetdm.com/guides/connect-end-user-to-wifi-with-certificate#smallstep).

### Labels page

> A new **Labels** page makes it easier to view and edit labels. You can find the new labels page in Fleet by heading to TODO

Author: ```suggestion
A new **Labels** page makes it easier to view and edit labels. You can find the new **Labels** page in Fleet by selecting your avatar in the top-right corner and selecting **Labels**.
```
